src/styles/buttons.scss

```diff
@@ -8,6 +8,11 @@ button,
   line-height: 33px;
   margin-bottom: 16px;
   font-family: Arial, Helvetica, sans-serif;
+  &[disabled] {
+    cursor: not-allowed;
+    pointer-events: all !important;
```

Comment:
do you mean auto here? what does this do

Reply:
I think that auto would still rely on the disabled attribute, which removes all pointer, including the cursor I wanted. This was recommended on a Bootstrap forum somewhere.

Come to think of it though, this might allow it to be clicked